python-packages/order_utils/src/zero_ex/order_utils/__init__.py

```diff
@@ -167,14 +249,15 @@ def generate_order_hash_hex(order: Order) -> str:
     ...         'takerFee': "0",
     ...         'expirationTimeSeconds': "12345",
     ...         'salt': "12345",
-    ...         'makerAssetData': "0x0000000000000000000000000000000000000000",
-    ...         'takerAssetData': "0x0000000000000000000000000000000000000000",
-    ...         'exchangeAddress': "0x0000000000000000000000000000000000000000",
+    ...         'makerAssetData': (0).to_bytes(1, byteorder='big') * 20,
+    ...         'takerAssetData': (0).to_bytes(1, byteorder='big') * 20,
```

Comment:
Can we return `(0).to_bytes(1, byteorder='big')` from a helper method too? Seems useful elsewhere.

Reply:
For all of the cases in this PR, I'd prefer to keep it this way, because these are "docstrings", example snippets that, in my opinion, should be as self-contained as possible, ideally not relying on any outside context at all.